src/modules/Elsa.Hosting.Management/Options/HeartbeatSettings.cs

```diff
@@ -0,0 +1,7 @@
+namespace Elsa.Hosting.Management.Options;
+
+public class HeartbeatSettings
+{
+    public TimeSpan InstanceHeartbeatRhythm { get; set; } = TimeSpan.FromMinutes(1);
```

Comment:
Although this is very descriptive, it has an unconventional sound to it. `HeartbeatInterval` sounds nicer and is simpler, too.